crates/nu-cli/src/completions/dotnu_completions.rs

```diff
@@ -58,7 +58,7 @@ impl Completer for DotNuCompletion {
             };
 
         // Check if the base_dir is a folder
-        if base_dir != "./" {
+        if base_dir != format!(".{}", SEP) {
```

Comment:
I'm not sure if there's a more "lightweight" to do it without the `format!`, please shout if there is.